doc/migrations/4.0.1_4.x.x.md

```diff
@@ -6,3 +6,46 @@ Currently, only the `urn:xmpp:http:upload:0` XMLNS is served.
 All major, modern client libraries and applications support the 0.3.0+ specification.
 If you experience any issues with making requests to the HTTP File Upload service, please update your client.
 
+## Retirement of old `*.cfg` format
+
+Since ____ release, we are no longer supporting `*.cfg` MongooseIM configuration format. For further application's usage, please use `TOML` format.
```

Comment:
I left it as a placeholder for upcoming TOMLed release version, once we clarify it.